internal/aws/ec2.go

```diff
@@ -175,6 +186,70 @@ func (c *Cloud) GetSecurityGroupByID(groupID string) (*ec2.SecurityGroup, error)
 	return securityGroups[0], nil
 }
 
+func (c *Cloud) ResolveSecurityGroupNames(ctx context.Context, in []string) ([]string, error) {
```

Comment:
I'd preferrer clear name like in -> sgIDOrNames   similar to the intOrString in k8s core.